### ISSUE TITLE
Missing space after links

### ISSUE THREAD

**Issue by wezm** [theiskaa/mdp#1]
There is a space missing after links. E.g.

```markdown
This [inline link](https://www.example.com/) is missing a space afterwards.
```

<img width="323" alt="Screenshot from 2024-10-10 09-41-46" src="https://github.com/user-attachments/assets/c0d9a851-1459-44f4-8932-5c574b6c6a14">

**Comment by theiskaa** [comment]
that's interesting. probably because markdown lexer handles the link then skips to the text without handling space. I'll check it out.

**Comment by theiskaa** [comment]
✓ Fixed in commit [21e79ba](https://github.com/theiskaa/mdp/commit/21e79baea6e05b8c9deae6244f12cda48f467641)

Please verify the changes and let me know if you encounter any issues.
